frontend/packages/console-app/src/components/volume-group-snapshot/_create-volume-group-snapshot.scss

```diff
@@ -0,0 +1,44 @@
+@import '../../../../../public/style/vars';
+@import '../../../../console-shared/src/styles/skeleton-screen.scss';
+
+.co-volume-snapshot__body {
+  display: inline-flex;
+}
+
+.co-volume-snapshot__form {
+  margin: var(--pf-t--global--spacer--md) 0;
+}
+
+.co-volume-snapshot__details-body {
+  margin-bottom: var(--pf-t--global--spacer--md);
+}
+
+.co-volume-snapshot__alert-body {
+  margin-top: var(--pf-t--global--spacer--md);
+}
+
+.co-volume-snapshot__info {
+  max-width: 50%;
+  margin-right: 0;
+
+  @media (max-width: $screen-sm-max) {
+    display: none;
+  }
+}
+
+.skeleton-activity {
+  animation: $skeleton-animation;
+  background: $skeleton-color;
+  opacity: 0;
+  border-radius: 6px;
+  margin: 0.5rem 0.7em 0.5rem 0.7em;
+  height: 30px;
+
+  &::after {
+    background-repeat: no-repeat;
+    content: '';
+    display: block;
+    height: 100%;
+    width: 100%;
+  }
+}
```

Comment:
why file name has `_` prefix ?? does other component files follow same convention ??

Reply:
Same convention followed in VolumeSnapshot